src/Ops.Controllers/Areas/SiteManagement/SegmentsController.cs

```diff
@@ -138,6 +142,12 @@ public async Task<IActionResult> Delete(IndexViewModel model)
         {
             try
             {
+                var episode = await _podcastService.GetEpisodeBySegmentIdAsync(model.Segment.Id);
+                if (episode != null)
+                {
+                    episode.ShowNotesSegmentId = null;
+                    await _podcastService.UpdatePodcastItemAsync(episode);
+                }
```

Comment:
Show notes segment deletion should be moved to the Podcasts controller, users without SiteManagement permissions won't be able to scroll through the segment index to find the segment to delete it. Instead there should be a delete notes icon next to the edit notes icon when viewing an episodes details.

With this change podcast segments should be added to the `GetSegmentInUseByAsync` function in SegmentService so podcasts segments aren't accidentally deleted from the segment page.

Reply:
Resolved in commit 00bd5d1